integrationTests/frontend/wallet/txInterception_test.go

```diff
@@ -39,11 +40,12 @@ func TestInterceptedTxWhithDataField(t *testing.T) {
 		0,
 		value,
 		"erd12dnfhej64s6c56ka369gkyj3hwv5ms0y5rxgsk2k7hkd2vuk7rvqxkalsa",
-		"erd1zq6l0g5yg6ehz7reatt3uhyfy89udx8q3p72fy5fhmfdvr883xgqufyvte",
-		"8bdaebfc67b5b7e8a325004b3934dc6f1f0329b3d29baa1763c663925384bd93f1de3a8ac297db918a233c13c4014431a8ed7455c03e2a706828a9a400b6b107",
+		"erd1xehvyp68ufu7gd543nrjs4m5fx2d090qkeqh5k7y3yt42en0ealszcr5n8",
+		"9fcccee9545345be4aa36883b5938581f9d1ff7eb54190b1507364d43aaa8236fe655ef75d77c8973779c321ff454cb35c6689947c81ebbc2208327b694a0708",
 		10,
 		100000,
-		[]byte("data@~`!@#$^&*()_=[]{};'<>?,./|<>><!!!!!"),
+		[]byte("erd-test-data"),
```

Comment:
you discarded the special characters. Does they still work?

Reply:
they work. I just generated something different for Mihai to test and forgot to add back that data field. Will be updated

Reply:
added again special characters